manifests/templates/reconciler-manager-configmap.yaml

```diff
@@ -267,9 +264,7 @@ data:
                k8s.pod.ip=$(KUBE_POD_IP),\
                k8s.node.name=$(KUBE_NODE_NAME),\
                k8s.deployment.name=$(KUBE_DEPLOYMENT_NAME),\
-               configsync.sync.kind=$(CONFIGSYNC_SYNC_KIND),\
-               configsync.sync.name=$(CONFIGSYNC_SYNC_NAME),\
-               configsync.sync.namespace=$(CONFIGSYNC_SYNC_NAMESPACE)"
+               k8s.pod.namespace=$(KUBE_POD_NAMESPACE)"
```

Comment:
this looks like a duplicate. There's already a k8s.pod.namespace on line 262

Reply:
Done